.github/workflows/autorelease.yml

```diff
@@ -89,7 +95,33 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           branch: ${{ github.base_ref }}
           tags: true
-          
+
+      - name: Set enviroment variables for release notes.
+        run: |
+          echo "RELEASE_TAG=$(git tag --sort=-taggerdate | head -1)" >> $GITHUB_ENV
+          echo "PREVIOUS_TAG=$(git tag --sort=-taggerdate | sed -n '2 p')" >> $GITHUB_ENV
+
+      - name: Create Release Notes
+        id: release_notes
+        run: |
+          # Scrape release notes for this release from the full changelog.
+          release_notes=$(./node_modules/.bin/auto-changelog --stdout | awk "/\[$RELEASE_TAG\]/{flag=1; next} /\[$PREVIOUS_TAG\]/{flag=0} flag")
```

Comment:
`auto-changelog` has a `--unreleased-only` flag that should allow you to generate the release notes without having to scrape them.

Reply:
I'll give this a try and see if it works... 
I was trying with other flags, like --starting-version, but those weren't providing the expected output.
Also tried 2-3 different Github actions for auto-generating release notes and struck out with those as well.

Reply:
@sherakama Looks like --unreleased-only gets the correct messages if I move it _before_ the npm version command:
![Screen Shot 2021-05-19 at 3 01 45 PM](https://user-images.githubusercontent.com/4481455/118884328-8847dd80-b8b3-11eb-9457-6a8114271c96.png)

However, we'd still need to do some post-processing to remove the additional lines at the beginning or pass in a custom template. 

Would you prefer one of those options, or should we stick with the existing solution?

Reply:
Thanks for looking into this. I don't want this to be more make work for you. I'll noodle on this a little more and if I don't come up with a compelling reason to change I think what you have is great.